### ISSUE TITLE
make 'pods blocked by from internet' low priority or out of scope or both

### ISSUE THREAD

**Issue by jayunit100** [jayunit100/network-policy-subproject#13]
```
- I want all pods to be blocked from accessing the internet.
```

```
I want all pods to be blocked from accessing the internet by default, 
but developers can add policies declaring what sites they need access to. 
The sites will be reachable, 
but if hackers break into the pod they can’t reach anywhere 
else besides those sites, except they can’t access 192.168.0.0/16 
even if they declare it.
```
- is more of a svc mesh thing.
- story needs to be clarified.

**Comment by jayunit100** [comment]
per Chris : `For the first part - Would blocking access to anything not declared in a CIDR egress policy not suffice?`

**Comment by jayunit100** [comment]
I think the user story implies that it should be easy to do this w/o CIDR ranges, so I updated that .
